README.md

```diff
@@ -101,6 +101,53 @@ scripts.
 If you would like to contribute your own demo, see our [demo submission
 guide](https://pennylane.ai/qml/demos_submission.html).
 
+## Build a PennyLane Docker image
+
+**Docker** support exists for building using **CPU** and **GPU** (Nvidia CUDA 11.1+) images.
+
+
+**Note:** Docker builds using "make" will work on Linux and MacOS only. For MS Windows you can use [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10).

```

Comment:
I've had some issues getting this to run on my M1 Mac, so I'd suggest adding a line about not supporting it for now (we could have a closer look at the issue in the future).

```suggestion
**Note:** Docker builds using "make" will work on Linux and MacOS only. For MS Windows you can use [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10). They are currently not supported on the Apple M1 chip (ARM64).

```

Reply:
@thisac  Interesting, I thought It will work on M1 as well.
Did you tried this?
https://docs.docker.com/docker-for-mac/apple-silicon/

Reply:
I can get it to run fine, but I'm having issues with the PennyLane installation step. There seems to be some required installations that, surprisingly, aren't there (`pkg-config` and `HDF5`, for example), and I couldn't get it to work simply by adding a step to install them. 

Unless you wish look into this a bit more, I'd suggest we leave this for now. Since everything seems to work for @mlxd, I'm happy to approve this with a note on M1 Mac support.